src/examples/java/LocalizationExample.java

```diff
@@ -0,0 +1,73 @@
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.JDABuilder;
+import net.dv8tion.jda.api.interactions.DiscordLocale;
+import net.dv8tion.jda.api.interactions.commands.Command;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.*;
+import net.dv8tion.jda.api.interactions.commands.localization.LocalizationFunction;
+import net.dv8tion.jda.api.interactions.commands.localization.ResourceBundleLocalizationFunction;
+
+import javax.security.auth.login.LoginException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class LocalizationExample
+{
+    public static void main(String[] args) throws LoginException, IOException
+    {
+        //The token is stored in "Config.txt" in the root folder of this project
+        final String token = Files.readAllLines(Paths.get("Config.txt")).get(0);
```

Comment:
I know this is technically better, but lets match the token usage in the other examples for consistency.